.circleci/config.yml

```diff
@@ -5,48 +5,100 @@ orbs:
   azure-cli: circleci/azure-cli@1.1.0
 
 jobs:
+  build:
+    docker: &python3-7
+      - image: circleci/python:3.7.9
+    steps:
+      - checkout
+      - run:
+          name: install pyodbc deps
+          command: |
+            sudo apt-get update && \
+            sudo apt-get install -y curl build-essential g++ apt-transport-https && \
```

Comment:
it would be nice to see if `sudo` is not needed (for all installs)

Reply:
I also would like to, but it seems CircleCi's python docker image doesn't give root user access for some reason